src/StickerService.ts

```diff
@@ -41,7 +41,10 @@ function buildWallpaperCss({
   .monaco-workbench .part.editor>.content .editor-group-container>.title .tabs-breadcrumbs .breadcrumbs-control,
   .ref-tree, /* find usages */
   .head, /* find usages */
-  .monaco-workbench .part.editor>.content .editor-group-container>.title .editor-actions
+  .monaco-workbench .part.editor>.content .editor-group-container>.title .editor-actions,
+  .editor-container, /* welcome screen */
+  .terminal-outer-container,  /* Terminal outer edge */
+  .markers-panel-container /* Problems */
```

Comment:
What was the problem with problems?

| Before | After |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/15972415/114385401-9a688a80-9b55-11eb-8bde-732aaa957fd4.png) | ![image](https://user-images.githubusercontent.com/15972415/114385485-b66c2c00-9b55-11eb-888d-36c1085d49e8.png) |